examples/scripts/README.rst

```diff
@@ -78,6 +78,8 @@ Miscellaneous
       Comparison of Optimization Algorithms for Total Variation Denoising
    `denoise_cplx_tv_pdhg.py <denoise_cplx_tv_pdhg.py>`_
       Complex Total Variation Denoising
+   `denoise_dncnn_with_noise_level_input.py <denoise_dncnn_with_noise_level_input.py>`_
+      Variants of DnCNNs for Image Denoising
```

Comment:
Comparison of DnCNN Variants for Image Denoising?

Reply:
I would also change the filename to something a bit shorter. Perhaps something like `denoise_dncnn_universal`?